### PR TITLE
ignore_run_exports from python

### PR DESCRIPTION
**Pull request by timsnyder-siv**
ignore_run_exports has to list the package that is injecting the run requirement, not the requirement that is being injected.

### CODE REVIEW

**Comment by timsnyder-siv** [comment]
@abejgonzalez , I reread the documentation for `ignore_run_exports` and think I see what I did wrong in #5.

**Review by abejgonzalez — APPROVED** [review]
LGTM. Once this is merged you can probably look directly at the package and see if the runtime requirement is still present

**Comment by timsnyder-siv** [comment]
@abejgonzalez I don't have write access to this repo and can't click the merge button.  Can you merge it?

**Comment by abejgonzalez** [comment]
Done. I also just added you as a maintainer.

**Pull request merged**
